docs/index.rst

```diff
@@ -24,88 +69,3 @@ A community platform for Big Data geoscience
-`Discourse forum <https://discourse.pangeo.io/>`_
-or browse our `GitHub repos <https://github.com/pangeo-data/>`_.
-
-
-.. raw:: html
-
-    <div class="row">
-     <div class="col-md-6 center-block">
-        <h2><a href="https://twitter.com/pangeo_data">Twitter Feed</a></h2>
-        <p class="text-center font-italic">We use twitter to share announcements and retweet content of interest to our community.</p>
-        <a class="twitter-timeline" data-width="100%" data-height="500" data-theme="light" data-link-color="#5eb130" href="https://twitter.com/pangeo_data?ref_src=twsrc%5Etfw">Tweets by pangeo_data</a> <script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script>
-     </div>
-     <div class="col-md-6 center-blockr">
-         <h2><a href="https://discourse.pangeo.io/">Discourse Forum</a></h2>
-         <p class="text-center font-italic">Discourse is a message board where we discuss all aspects of the project in a friendly, inclusive way.</p>
-         <div class="feedgrabbr_widget" id="fgid_d0649815dd692ed59cba2d8d0"></div>
-         <script>if (typeof (fg_widgets) === "undefined") fg_widgets = new Array(); fg_widgets.push("fgid_d0649815dd692ed59cba2d8d0");</script>
-         <script async src="https://www.feedgrabbr.com/widget/fgwidget.js"></script>
-     </div>
-    </div>
-
-What is Pangeo?
-===============
-
-Pangeo is first and foremost a community promoting open, reproducible, and scalable science.
-This community provides documentation, develops and maintains software, and deploys
-computing infrastructure to make scientific research and programming easier.
-The Pangeo software ecosystem involves
-open source tools such as xarray, iris, dask, jupyter, and many other packages.
-There is no single software package called "pangeo"; rather, the Pangeo project serves
-as a coordination point between scientists, software, and computing infrastructure.
-On this website, scientists can find guides for accessing data and performing analysis
-using these tools (read the :ref:`quickstart`, browse the :ref:`use-cases`, and
-learn about the :ref:`packages`).
-Those interested in building infrastructure can find
-instructions for deploying Pangeo environments on HPC or cloud clusters
-(learn about the :ref:`architecture` or read the :ref:`setup-guides`).
-For more general information, read :ref:`about`, see the :ref:`collaborators`,
-or read the :ref:`FAQ`.
-Welcome to the Pangeo community!
-
-Our Goals
-^^^^^^^^^
-
-#. Foster collaboration around the open source scientific python ecosystem for
-   ocean / atmosphere / land / climate science.
-#. Support the development with domain-specific geoscience packages.
-#. Improve scalability of these tools to handle petabyte-scale datasets on
-   HPC and cloud platforms.
-
-Pangeo Gallery
-^^^^^^^^^^^^^^
-
-Science applications and detailed tutorial for using Pangeo can be found in
-`Pangeo Gallery <http://gallery.pangeo.io>`_.
```

Comment:
I think this is important to have a Pangeo Gallery link somewhere in the homepage. Maybe in 'learn more' table section?